### ISSUE TITLE
When we use updateactivity method, "AI generated" is not displaying 

### ISSUE THREAD

**Issue by SubbaReddi** [MicrosoftDocs/msteams-docs#11435]
### Type of issue

Missing information

### Feedback

When send message using SendActivityAsync, we see "AI generated" label as shown in below screenshot.
![image](https://github.com/user-attachments/assets/fe078260-e384-47b3-9962-6ef599fd60ef)

But if we update message using UpdateActivityAsync, label is not displaying.
![image](https://github.com/user-attachments/assets/77612bd6-5ebb-417e-a59c-9df92116274e)

As it is known behavior for now, adding note will help.

### Page URL

https://learn.microsoft.com/en-us/microsoftteams/platform/bots/how-to/bot-messages-ai-generated-content?tabs=before%2Cbotmessage

### Content source URL

https://github.com/MicrosoftDocs/msteams-docs/blob/main/msteams-platform/bots/how-to/bot-messages-ai-generated-content.md

### Author

@surbhigupta

### Document Id

605a8486-6e6a-31d5-e8a1-021691206099

**Comment by Nivedipa-MSFT** [comment]
@SubbaReddi - Thanks for reporting your issue. We will check this and update you soon.

**Comment by pedrojlazevedo** [comment]
Hello all, this is happening to me too.

My attemps were:
```javascript
                                    await context.updateActivity({
                                        id: activityId,
                                        type: 'message',
                                        attachments: [adaptiveCard],
                                        entities: [
                                            {
                                                type: "https://schema.org/Message",
                                                "@type": "Message",
                                                "@context": "https://schema.org",
                                                additionalType: ["AIGeneratedContent"],
                                            }
                                        ]
                                    });
```` 

And also:
```javascript
const newActivity = MessageFactory.attachment(CardFactory.adaptiveCard(newAdaptiveCard));
                                newActivity.entities = [
                                    {
                                        type: "https://schema.org/Message",
                                        "@type": "Message",
                                        "@context": "https://schema.org",
                                        additionalType: ["AIGeneratedContent"],
                                    }
                                ]
                                newActivity.id = activityId;
                                await context.updateActivity(newActivity);
```

Is this an expected behaviour? 
Is the "Edited" message replacing the AI Content Generator tag?

Thank you in advance.

**Comment by v-ramyas5** [comment]
Hello @Nivedipa-MSFT, we are working on the PR. Will update once the PR is merged into live.
